tests/views/test_message.py

```diff
@@ -509,6 +562,16 @@ def test_form_submit_with_FDI_data(self, mock_request, mock_get_jwt):
         mock_request.get(url_get_threads_list, json=thread_list, status_code=200)
```

Comment:
Do you still need these old mocks if you're not going to the inbox page anymore?  I don't think you do, I'd try and remove some of these mock requests to narrow it down to only what you need